src/boco_core/boco_input/Input_handling.odin

```diff
@@ -0,0 +1,49 @@
+package boco_input
+
+import "core:log"
+import "../boco_event_data"
+
+Input_event :: struct{
```

Comment:
Was thinking about turning this into a union but not sure

Reply:
If the Input_event can never be both a key_event and a mouse_event then yeah union all the way.